frontend/src/index.test.js

```diff
@@ -1,37 +1,51 @@
-import { render, screen } from "@testing-library/react";
-import {setupLandingPage} from "./setupTests";
+// TODO: Test cases for: makeGraph(), setDescendantStats(), getIndentedTreeNodes()?, cset comparison page line 144?
```

Comment:
## TODO: Test cases
- [ ] `makeGraph()`
- [ ] `setDescendantStats()`
- [ ] `getIndentedTreeNodes()`?
- [ ] cset comparison page line 144?
  - I don't know if this is the same as one of the above but at last meeting with Siggie I remember this being somehow different